cristin-person/src/main/java/no/unit/nva/cristin/person/client/AuthorizedCristinPersonApiClient.java

```diff
@@ -0,0 +1,55 @@
+package no.unit.nva.cristin.person.client;
+
+import no.unit.nva.cristin.person.model.cristin.CristinPerson;
+import no.unit.nva.utils.UriUtils;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.NotFoundException;
+import nva.commons.core.paths.UriWrapper;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpResponse;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static no.unit.nva.cristin.common.ErrorMessages.ERROR_MESSAGE_IDENTIFIER_NOT_FOUND_FOR_URI;
+import static no.unit.nva.cristin.model.Constants.CRISTIN_API_URL;
+import static no.unit.nva.cristin.person.client.CristinPersonQuery.CRISTIN_API_PERSONS_PATH;
+import static no.unit.nva.utils.UriUtils.PERSON;
+
+public class AuthorizedCristinPersonApiClient extends CristinPersonApiClient {
```

Comment:
I am not sure whether this client should be called Authorized. The only thing that makes it authorized is the fact that the HttpClient is authorized.